example/src/main/java/com/spruceid/mobilesdkexample/wallet/HandleOID4VPView.kt

```diff
@@ -0,0 +1,123 @@
+package com.spruceid.mobilesdkexample.wallet
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.spruceid.mobile.sdk.rs.Holder
+import com.spruceid.mobile.sdk.rs.ParsedCredential
+import com.spruceid.mobile.sdk.rs.PermissionRequest
+import com.spruceid.mobile.sdk.rs.SdJwt
+import com.spruceid.mobilesdkexample.ui.theme.Inter
+import com.spruceid.mobilesdkexample.ui.theme.MobileSdkTheme
+import com.spruceid.mobilesdkexample.viewmodels.IRawCredentialsViewModel
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.Button
+
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalPermissionsApi::class)
+@Composable
+fun HandleOID4VPView(
+    navController: NavController,
+    rawCredentialsViewModel: IRawCredentialsViewModel,
+    url: String
+) {
+    val scope = rememberCoroutineScope()
+
+    val rawCredentials by rawCredentialsViewModel.rawCredentials.collectAsState()
+
+    var holder by remember { mutableStateOf<Holder?>(null) }
+    var permissionRequest by remember { mutableStateOf<PermissionRequest?>(null) }
+
+    LaunchedEffect(Unit) {
+        println("URL: $url")
+
+        val credentials = rawCredentials.map { rawCredential ->
+            ParsedCredential
+                // TODO: Update to use VDC collection in the future
+                // to detect the type of credential.
+                .newSdJwt(SdJwt.newFromCompactSdJwt(rawCredential.rawCredential))
+                .intoGenericForm()
+        }
+
+        withContext(Dispatchers.IO) {
+
+            // Retrieve the permission request from the OID4VP URL.
+            val trustedDids = MutableList(1, { list ->
+                "did:web:192.168.68.72%3A3000:oid4vp:client"
```

Comment:
Move trustedDID to utils file for configuration.

NOTE: do wildcards work for trusted dids? (for testing)